README.md

```diff
@@ -258,21 +258,590 @@ As part of the deployment, you'll need to set up [git integration](https://rasa.
 
 You will need to have docker installed in order to build the action server image. If you haven't made any changes to the action code, you can also use the [public image on Dockerhub](https://hub.docker.com/r/rasa/financial-demo) instead of building it yourself.
 
+Build & tag the image:
 
-See the Dockerfile for what is included in the action server image,
+```bash
+export ACTION_SERVER_DOCKERPATH=<dockerID>/<name-of-image>:<tag-of-image>
+make docker-build
+```
+
+Run the action server container:
+
+```bash
+make docker-run
+```
 
-To build the image:
+Perform a smoke test on the health endpoint:
 
 ```bash
-docker build . -t <name of your custom image>:<tag of your custom image>
+make docker-test
 ```
 
-To test the container locally, you can then run the action server container with:
+Once you have confirmed that the container works as it should, push the container image to a registry:
 
 ```bash
-docker run -p 5055:5055 <name of your custom image>:<tag of your custom image>
+# login to a container registry with your credentials
+docker login  
+
+# check the registry logged into
+docker system info | grep Registry
+
+# push the action server image
+make docker-push
 ```
 
-Once you have confirmed that the container works as it should, you can push the container image to a registry with `docker push`
+# CI/CD
+
+## Summary
+
+A CI/CD pipeline is used to test, build and deploy the financial-demo bot to AWS EKS:
```

Comment:
I think it'd make sense to link the docs page here: https://rasa.com/docs/rasa/setting-up-ci-cd

Reply:
Done!